tests/playbooks/tests_802_1x.yml

```diff
@@ -88,7 +100,89 @@
+          vars:
+            network_connections:
+              - name: "{{ interface }}"
+                persistent_state: absent
+                state: absent
+        - name: >-
+            TEST: 802.1x profile with unencrypted private key and ca_path
+          debug:
+            msg: "##################################################"
+        - name: Create directory for ca_path test
+          file:
+            path: "/etc/pki/tls/my_ca_certs"
+            state: directory
+            mode: 0755
+        - name: Copy cacert to ca_path
+          copy:
+            src: "cacert.pem"
+            dest: "/etc/pki/tls/my_ca_certs/cacert.pem"
+            mode: 0644
+        - name: Hash cacert
+          command: openssl x509 -hash -noout
+            -in /etc/pki/tls/my_ca_certs/cacert.pem
+          register: cacert_hash
+        - name: Add symlink for cacert
+          file:
+            state: link
+            path: "/etc/pki/tls/my_ca_certs/{{ cacert_hash.stdout }}.0"
+            src: cacert.pem
+        - name: Get NetworkManager version
+          command:
+            cmd: rpm -qa NetworkManager
+            warn: false
+          register: __network_NM_NVR
+        - block:
+            - import_role:
+                name: linux-system-roles.network
+              vars:
+                network_connections:
+                  - name: "{{ interface }}"
+                    interface_name: veth2
+                    state: up
+                    type: ethernet
+                    ip:
+                      address:
+                        - 203.0.113.2/24
+                      dhcp4: "no"
+                      auto6: "no"
+                    ieee802_1x:
+                      identity: myhost_capath
+                      eap: tls
+                      private_key: /etc/pki/tls/client.key.nocrypt
+                      client_cert: /etc/pki/tls/client.pem
+                      private_key_password_flags:
+                        - not-required
+                      ca_path: /etc/pki/tls/my_ca_certs
+            - name: trigger failure in case the role did not fail
+              fail:
+                msg: after test
+              when:
+                - __network_NM_NVR.stdout in __NM_capath_ignored_NVRs
+            - name: "TEST: I can ping the EAP server"
+              command: ping -c1 203.0.113.1
+          rescue:
+            - name: Check that role failed as expected
+              fail:
+                msg: "ca_path test did not fail but
+                    {{ __network_NM_NVR.stdout }} known to be broken.
+                    Investigate or update the versions."
+              when:
+                - __network_NM_NVR.stdout in __NM_capath_ignored_NVRs
+                - ansible_failed_result.msg == 'after test'
+            - name: Check if role failed unexpectedly
+              assert:
+                that:
+                  - ansible_failed_result.msg != 'after test'
```

Comment:
a list entry marker `- ` seems to be missing on the line below. Or maybe you meant `fail_msg` (an option for `assert`) instead of
```yaml
fail:
  msg:
```
(a separate `fail` task).

Reply:
I think I meant to remove the fail and use the msg for the assert.